replicatedhq/ship
# Set up dependencies for Helm binary embed

What I Did
------------
Update `Gopkg.toml` with overrides that provides the necessary subdependencies for Helm to be integrated into the Ship binary.

How I Did it
------------
- Add Kubernetes 1.10.6 through an override (version required by Helm)
- Add apimachinery version where first version of `MergeStrategicMergeMapPatchUsingLookupPatchMeta` appeared (we use this as part of our Kustomize patching process)

How to verify it
------------
Binary should continue to build with `make bin/ship` after running `dep ensure`.

Description for the Changelog
------------
N/A


Picture of a Boat (not required but encouraged)
------------
![](https://vignette.wikia.nocookie.net/avatar/images/9/9b/The_Zhu_Li.png/revision/latest?cb=20140528115802)
